### PR TITLE
Timezone selector added

### PR DESCRIPTION
**Pull request by JamesSCBD**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "luxon": "1.25.0",
     "nuxt": "2.14.6",
     "nuxt-fontawesome": "0.4.0",
+    "tzdata": "1.0.25",
```
Comment (sbilodeau) [inline package.json:39]: exceptionally  use ~ in version so timezones will upgrade on deploy (last number, 25, 26, 27...)

```suggestion
    "tzdata": "~1.0.25",
```

**Review by sbilodeau — CHANGES_REQUESTED** [review]

File: components/controls/selectors/TimezoneSelector.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      type   : String,
+      default: 'Select timezone',
+    },
+    disabled: {
+      type   : Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    selectedTimezone: { get() { return this.value; }, set(value) { this.$emit('input', (value || '')); } },
```
[the remark this answers is not in the document]
Comment (JamesSCBD) [reply components/controls/selectors/TimezoneSelector.vue]: DONE!

File: components/controls/selectors/TimezoneSelector.vue [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import tzdata from 'tzdata';
+
+export default {
+  name      : 'TimezoneSelector',
+  components: { Multiselect },
+  props     : {
+    value: {
+      type    : String,
+      required: true,
+      default : '',
```
[the remark this answers is not in the document]
Comment (JamesSCBD) [reply components/controls/selectors/TimezoneSelector.vue]: DONE!

File: components/controls/selectors/TimezoneSelector.vue [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      default: 'Select timezone',
+    },
+    disabled: {
+      type   : Boolean,
+      default: false,
+    },
+  },
+  computed: {
+    selectedTimezone: { get() { return this.value; }, set(value) { this.$emit('input', (value || '')); } },
+    timezoneOptions() { return tzdata ? Object.keys(tzdata.zones) : []; },
```
[the remark this answers is not in the document]
Comment (JamesSCBD) [reply components/controls/selectors/TimezoneSelector.vue]: DONE!

[the remark this answers is not in the document]
Comment (JamesSCBD) [reply components/controls/selectors/TimezoneSelector.vue]: DONE!

File: package.json (hunk 1 above)
Comment (JamesSCBD) [reply package.json]: DONE!

**Review by sbilodeau — APPROVED** [review]

**Pull request merged**